frontends/concrete-python/concrete/fhe/representation/graph.py

```diff
@@ -454,7 +454,10 @@ def format(
                 bounds += "∈ ["
 
                 lower, upper = node.bounds
-                assert type(lower) == type(upper)  # pylint: disable=unidiomatic-typecheck
+
+                # pylint: disable=unidiomatic-typecheck
```

Comment:
was unsure how to fix properly

Reply:
is disable locally, should be fine for now

Reply:
If you really want to do it like this, sure. But I'd still disable it for the file in `.ruff.toml`

Reply:
it's just once, why disabling it in the file?

Reply:
Sorry it was meant to be for the assign bit widths processor, so the conversation above.